lambda/custom/helperFunctions.js

```diff
@@ -163,20 +164,14 @@ const channelUnreadMessages = async (channelName, unreadCount, headers) =>
 					const msgs = [];
 
 					for (let i = 0; i <= unreadCount - 1; i++) {
-						msgs.push(
-							ri('GET_UNREAD_MESSAGES_FROM_CHANNEL.MESSAGE', {
-								name: res.messages[i].u.username,
-								message: res.messages[i].msg,
-							})
-						);
+						msgs.push(`${res.messages[i].u.username} says, ${res.messages[i].msg} <break time="0.7s"/> `);
```

Comment:
"says" word will not be localized here. A minor issue, for now, but should definitely add it to the list.

Reply:
> "says" word will not be localized here. A minor issue, for now, but should definitely add it to the list.

Sure